src/cli/index.ts

```diff
@@ -0,0 +1,52 @@
+const minimist = require('minimist');
+const argv = minimist(process.argv.slice(2));
+
+import * as request from 'request';
```

Comment:
move `request` outta main entry index file. create a new file `utils.ts` or named it like you think is the best, then export function that returns a `Promise`. this was done quick & dirty, why did you use callback directly and making http request in main entry point?